vaadin-details-flow-demo/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>vaadin-details-flow-parent</artifactId>
-        <version>2.0-SNAPSHOT</version>
+        <version>2.1-SNAPSHOT</version>
```

Comment:
Nit: I guess we can bump notification snapshot below as well.

Reply:
Done.